docs/velocity/dev/how-to/porting-from-velocity-1.mdx

```diff
@@ -20,7 +20,7 @@ introduced in Velocity 1.1.0.
 
 `toml4j`, deprecated in Velocity 1.1.0 (as it is no longer maintained), has not been removed to
 provide more time for plugins to migrate to Configurate 3. However, you should prepare to either
-switch to Configurate 3 or shade toml4j into your plugin directly.
+switch to Configurate 3 or shade TOML4J into your plugin directly.
```

Comment:
This should probably mention Configurate 4 now? Since velocity 3.3.x Configurate 4 is bundled

Reply:
the version could probably be removed altogether